local_pathfinding/node_navigate.py

```diff
@@ -67,15 +77,23 @@ def __init__(self):
             qos_profile=10,
         )
 
-        # publishers and their timers
+        # publishers
         self.desired_heading_pub = self.create_publisher(
             msg_type=DesiredHeading, topic="desired_heading", qos_profile=10
         )
+        self.lpath_data_pub = self.create_publisher(
+            msg_type=LPathData, topic="local_path", qos_profile=10
+        )
+
+        # publisher timers
         pub_period_sec = self.get_parameter("pub_period_sec").get_parameter_value().double_value
         self.get_logger().debug(f"Got parameter: {pub_period_sec=}")
         self.desired_heading_timer = self.create_timer(
             timer_period_sec=pub_period_sec, callback=self.desired_heading_callback
         )
+        self.lpath_data_timer = self.create_timer(
+            timer_period_sec=pub_period_sec, callback=self.lpath_data_callback
+        )
```

Comment:
I think we should publish lpath_data at a lower frequency

Reply:
At what frequency should we publish lpath_data

Reply:
Actually sorry can you revert this to `pub_period_sec`, let's let NET determine how often to write to the DB rather than us determining it for us